packages/expo-development-client/bundle/App.tsx

```diff
@@ -8,16 +8,17 @@ import {
   SafeAreaView,
   StatusBar,
   StyleSheet,
-  Button,
   TextInput,
+  Platform,
+  TouchableOpacity,
 } from 'react-native';
 
 const DevelopmentClient = NativeModules.EXDevelopmentClient;
 
 // Use development client native module to load app at given URL, notifying of
 // errors
 
-const loadAppFromUrl = async (url: string) => {
+const loadAppFromUrl = async (url: string): bool => {
```

Comment:
Won't be easier to pass the `setLoading` function to `loadAppFromUrl`?

Reply:
Good point, I'll change it to be that way.